### PR TITLE
Mark 'lh' and 'rlh' units as standard compliant

### PR DESCRIPTION
**Pull request by Th3S4mur41**
#### Summary

'lh' and 'rlh' are part of the CSS Values and Units Module Level 4 draft just like small, large, and dynamic viewport units.

They should therefor also be marked as following the standard track
Both lengths were added as [supported in Chromium 109 in a previous PR](https://github.com/mdn/browser-compat-data/pull/18624)

#### Test results and supporting details

https://www.w3.org/TR/css-values-4/#lh
https://www.w3.org/TR/css-values-4/#rlh

### CODE REVIEW

File: css/types/length.json [hunk 1]
```diff
@@ -290,8 +290,8 @@
               "webview_android": "mirror"
             },
             "status": {
-              "experimental": true,
-              "standard_track": false,
+              "experimental": false,
```
Comment (queengooborg) [inline css/types/length.json:293]: We do not mark "experimental" based upon spec status, but rather based upon browser support.

```suggestion
              "experimental": true,
```

File: css/types/length.json [hunk 2]
```diff
@@ -368,8 +368,8 @@
...
```
Comment (queengooborg) [inline css/types/length.json:371]:
```suggestion
              "experimental": true,
```

**Review by queengooborg — CHANGES_REQUESTED** [review]
Along with the below changes, could you also add those URLs as `spec_url` properties?

**Comment by Th3S4mur41** [comment]
Spec URL added 😊

**Review by queengooborg — APPROVED** [review]
Thanks, looking good!

**Pull request merged**